cmd/util/flags/configflags/ipfs.go

```diff
@@ -21,6 +21,8 @@ var IPFSFlags = []Definition{
 		DefaultValue:         Default.Node.IPFS.SwarmKeyPath,
 		Description:          "Optional IPFS swarm key required to connect to a private IPFS swarm",
 		EnvironmentVariables: []string{"BACALHAU_IPFS_SWARM_KEY"},
+		Deprecated:           true,
+		DeprecatedMessage:    ipfsEmbeddedDeprecationMessage,
```

Comment:
Where's this message?

Reply:
line 5 of the file. It's not in the diff as it was added by Ross a while back when he was part way through some of the deprecation work.